index.js

```diff
@@ -265,13 +265,15 @@ Speech.prototype.phoneme = function (alphabet, ph, word) {
  * @returns {*}
  * @private
  */
-Speech.prototype._escape = function (word) {
+Speech.prototype._escape = function (word, allowTags) {
     if (typeof(word) === "string") {
```

Comment:
Let’s leave the escape function alone and assume that SSML provide in the sayWithSSML is valid and escaped XML.

Reply:
cool - updated